desc/perturbations.py

```diff
@@ -120,6 +123,7 @@ def perturb(
     if verbose > 0:
         print("Factorizing linear constraints")
     timer.start("linear constraint factorize")
+    # FIXME: linting/editor picks up that not enough values are unpacked here
```

Comment:
is this still an issue? Might be that there's too many? often linters complain if a function has too many returns values

Reply:
Weird my editor doesn't pick it up anymore. Maybe jetbrains fixed a bug.